packages/babel-parser/src/parser/expression.js

```diff
@@ -1460,6 +1460,26 @@ export default class ExpressionParser extends LValParser {
     return this.finishNode(node, "TemplateLiteral");
   }
 
+  parseKeyValuePairs() {
+    const propHash: any = Object.create(null);
+    const node = this.startNode();
+    node.properties = [];
+
+    do {
+      const prop = this.parseObjectMember();
```

Comment:
It maybe overkill to use `parseObjectMember` here because it is _likely_ to include only primitive types (see https://github.com/tc39/proposal-module-attributes/issues/22#issuecomment-558754321), i.e.
```js
import foo from "foo" with loaders: ["bar", "qux"]
```
are not allowed because array is not a primitive type. As a stage-1 proposal, I suggest we implement the minimal superset of proposal. Should the proposal accepts more types, we can always drops restrictions later.

Reply:
Understood. I will write a simple parseKeyValue function. Thank you 👍

Reply:
Refactored with the suggested changes

Reply:
We aren't sure if we want to allow any value there, let's keep it simple for now I think; only stringliterals

Reply:
Made the changes and now we are only allowing string literals.